pytorch/serving/model-store/rest-test.sh

```diff
@@ -33,4 +33,4 @@ curl --fail -O https://raw.githubusercontent.com/pytorch/serve/master/docs/image
 curl --fail -X GET http://localhost:8080/ping
 curl --fail -X POST "http://localhost:8081/models?initial_workers=1&synchronous=true&url=/home/model-server/model-store/squeezenet1_1.mar&model_name=squeezenet"
 curl --fail -X POST http://127.0.0.1:8080/v2/models/squeezenet/infer -T /home/model-server/model-store/kitten_small.jpg
-rm -rf /home/model-server/model-store/squeezenet1_1.mar kitten_small.jpg
```

Comment:
We should cleanup this file

Reply:
It will make the tests after it fail since this directory, and in turn the file, is mounted and all other following tests require the file. One way we can avoid it if we download it using curl command but then we won't be using the file that's generated from the model-archiving test.

Reply:
ipex-serving-grpc-inference is the only test remaining, and it uses densenet.

Can we make sure this file gets deleted at some point?

Reply:
`rest-test.sh` is used in 3rd test as well: [Reference](https://github.com/intel/ai-containers/blob/3e4b6e72822e3a957bff61e55c3e87b50ca7ce32/pytorch/serving/tests.yaml#L39). Which means the test after will fail. You're right about the grpc test though. So only 1 test will be affected.

Reply:
that is a different file, it's the test in the workflows directory

Reply:
Updated.